src/app/ad-authoring/ad-authoring.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="ad-authoring">
+    <input type="text" #landingUrl placeholder="Landing Page URL"/><br>
+    <input type="text" #landingType placeholder="Landing Page Type"/><br>
+    <select #callToAction>
+        <option value="Shop Now">Shop Now</option>
+        <option value="Apply Now">Apply Now</option>
+        <option value="Book Now">Book Now</option>
```

Comment:
We should populate these options using an *ngFor enumerating all the CTA enum values. We can go ahead and define a CTA enum for all the valid CTA values, and define a mapping of CTA enum to display value to use here.